roles/foreman_installer_devel_scenario/files/katello-devel.yaml

```diff
@@ -5,8 +5,6 @@
 :log_name: katello-devel.log
 :log_level: DEBUG
 :no_prefix: false
-# Forklift always overwrites the answers so let's not pretend we save them
```

Comment:
@wbclark is this no longer true?

Reply:
@ehelms with this change, yes the comment becomes no longer true, so we can really save the scenario answers now

`ansible.builtin.copy` module has a `force` parameter which defaults to `yes` -- see https://docs.ansible.com/ansible/latest/collections/ansible/builtin/copy_module.html#parameter-force

This PR explicitly sets `force: no` so that the devel installer role creates the devel scenario answers file if it doesn't already exist, but it will no longer overwrite the file if it has changes.

My assumption here is that if the developer _wants_ to reset to the initial state, a better way is to just destroy and recreate the box.

So in summary, the commit introduces two changes:

1. Don't overwrite the devel scenario answers file if we re-run the playbook

2. Do save config changes in the devel scenario answers file, so the installer can be used to make persistent changes (the same way we already use it in all the non-devel scenarios)